java/CHANGELOG.md

```diff
@@ -4,5 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```

Comment:
I guess we should make a tag after merging? So this can be `0.1.1`

Reply:
yeah sure